### PR TITLE
Fix broken gallery links - RSC 1074

### PR DESCRIPTION
**Pull request by achen5**
https://issues.sonatype.org/browse/RSC-1074

### CODE REVIEW

File: gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx [hunk 1]
```diff
@@ -17,15 +17,15 @@ function NxDropdownActionsExample() {
...
         <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
         <span>Go somewhere</span>
       </a>
-      <a href="#/" className="nx-dropdown-link">
+      <a href="https://www.sonatype.com/" className="nx-dropdown-link">
         <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
         <span>Go somewhere else</span>
       </a>
-      <a href="#/" className="nx-dropdown-link">
+      <a href="https://developer.mozilla.org/en-US/docs/Web" className="nx-dropdown-link">
```
Comment (rpokorny) [inline gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx:28]: Why these changes? `#/` is valid, it's our home page

File: gallery/src/components/NxIconDropdown/NxIconDropdownPage.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const NxIconDropdownPage = () =>
       <NxP>
         A variation of the dropdown menu component which uses a single icon as its toggle.
         <NxCode>NxIconDropdown</NxCode> shares all of the same props as{' '}
-        <NxTextLink href="#/pages/NxDropdown" className="nx-dropdown-button">
+        <NxTextLink href="#/pages/Dropdown" className="nx-dropdown-button">
```
Comment (rpokorny) [inline gallery/src/components/NxIconDropdown/NxIconDropdownPage.tsx:25]: The `.nx-dropdown-button` class shouldn't be here, let's go ahead and remove that

File: gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx [hunk 3]
```diff
@@ -22,7 +22,8 @@ const NxStatefulTreeViewRadioSelectPage = () =>
         to <NxCode>NxStatefulCollapsibleRadioSelect</NxCode> and
         is now deprecated and will be removed in the next major version.
         Please use
-        {' '}<NxTextLink href="#/pages/NxStatefulCollapsibleRadioSelect">NxStatefulCollapsibleRadioSelect</NxTextLink>
+        {' '}<NxTextLink href="#/pages/Stateful%20Collapsible%20Radio-Select">NxStatefulCollapsibleRadioSelect
```
Comment (rpokorny) [inline gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx:25]: IMO the cleanest formatting here would be to put the opening tag on its own line, then the content on its own line (indented), and then the closing tag on its own line like it already is

File: gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx (hunk 1 above)
Comment (achen5) [reply gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx:28]: > Why these changes? `#/` is valid, it's our home page

If you look at `/#/pages/Dropdown`, at the `Links Example`, each link has the external link icon to the left of it, so I changed the links from the internal `#/` to different external links.

Comment (rpokorny) [reply gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx:28]: > If you look at /#/pages/Dropdown, at the Links Example, each link has the external link icon to the left of it, so I changed the links from the internal #/ to different external links.

Hmm so they do.  But the CSS classes aren't set up to imply that these links are specifically external, _all_ dropdown menu links have this icon.  Which makes me wonder if that's supposed to be the case.  Checking Kaleido, I don't see mockups for dropdown menu links at all.  I think we need to talk to the designers about that.  For now though I guess your change is fine

**Comment by rpokorny** [comment]
What system did you run the visual tests on? I'm seeing a whole bunch of font rendering updates that suggest you used a different operating system or something.  Unfortunately, the visual tests are quite sensitive to that sort of thing.  That is why when updating the visual test snapshots you must use the docker container as described in the Readme (or use Linux directly)

**Comment by achen5** [comment]
> What system did you run the visual tests on? I'm seeing a whole bunch of font rendering updates that suggest you used a different operating system or something. Unfortunately, the visual tests are quite sensitive to that sort of thing. That is why when updating the visual test snapshots you must use the docker container as described in the Readme (or use Linux directly)

Ran and updated visual test snapshots in docker container. Ready for review

File: gallery/src/components/NxIconDropdown/NxIconDropdownPage.tsx (hunk 2 above)
Comment (achen5) [reply gallery/src/components/NxIconDropdown/NxIconDropdownPage.tsx]: > The `.nx-dropdown-button` class shouldn't be here, let's go ahead and remove that

Fixed here [1ebf468](https://github.com/sonatype/sonatype-react-shared-components/pull/721/commits/1ebf468a967985ef012b09748741ff63da0576c5)

File: gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx [hunk 4]
```diff
@@ -22,7 +22,10 @@ const NxStatefulTreeViewRadioSelectPage = () =>
         to <NxCode>NxStatefulCollapsibleRadioSelect</NxCode> and
         is now deprecated and will be removed in the next major version.
         Please use
-        {' '}<NxTextLink href="#/pages/NxStatefulCollapsibleRadioSelect">NxStatefulCollapsibleRadioSelect</NxTextLink>
+        {' '}
```
Comment (rpokorny) [inline gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx:25]: Minor: move this to previous line

**Review by rpokorny — APPROVED** [review]

File: gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx (hunk 4 above)
Comment (achen5) [reply gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx]: Fixed: [8e9fa45](https://github.com/sonatype/sonatype-react-shared-components/pull/721/commits/8e9fa45236f867c20f079aaaacaf49c4db324b9e)
